src/Compilers/Core/Portable/MetadataReader/PEModule.cs

```diff
@@ -2822,7 +2822,7 @@ private ConstantValue GetConstantValueOrThrow(ConstantHandle handle)
             return ConstantValue.Bad;
         }
 
-        internal AssemblyReferenceHandle GetAssemblyForForwardedType(string fullName, bool ignoreCase, out string matchedName)
+        internal HashSet<AssemblyReferenceHandle> GetAssemblyRefsForForwardedType(string fullName, bool ignoreCase, out string matchedName)
```

Comment:
Perhaps we should return a tuple instead of a HasSet, we probably are never interested in more than two references.